src/services/cacheConfigUtils.ts

```diff
@@ -146,4 +147,132 @@ export class CacheConfigUtils {
 
     return contentTypes;
   }
+
+  public static createCacheConfigFromData(data: CacheConfiguration): string {
+    let cache;
+
+    if (data.basic.topology === 'Distributed') {
+      cache = {
+        'distributed-cache': {
```

Comment:
Storage (heap or off_heap) is not added in the configuration yet. Not sure about that, because I saw `"storage" : "local-heap"` also in the documentation.

Reply:
@oraNod might help more on this

Reply:
@dipguptaredhat `local-heap` is different. That configures how Infinispan stores the index for a cache. It stores the index in the local JVM heap instead of somewhere persistent so it's volatile. 

For cache configuration, we should only be concerned with heap / off-heap. This is where Infinispan stores entries in the cache, either in the JVM heap (managed memory space) or off-heap (native, unmanaged memory).

Note that it's always "off-heap" with a hyphen. If you haven't seen it you can find that in the terminology guide at: https://infinispan.org/docs/stable/titles/contributing/contributing.html#terminology